styles/style.css

```diff
@@ -0,0 +1,62 @@
+/* 
+
+nav {
+    display: inline;
+    float: right
+} */
```

Comment:
As a style preference, I recommend getting rid of commented code wherever possible. But if there is some reason why you want the commented code to stick around, there should be a comment explaining what that reason is.